bundles/ch.elexis.core/model/model.ecore

```diff
@@ -160,6 +160,11 @@
     <eOperations name="removeRole">
       <eParameters name="role" lowerBound="1" eType="#//IRole"/>
     </eOperations>
+    <eOperations name="login" eType="#//IUser">
+      <eParameters name="username" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
+      <eParameters name="password" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
+    </eOperations>
+    <eOperations name="isRemote" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EBoolean"/>
```

Comment:
siehe kommentar oben - brauchen wir das? Wir sollten eher diskutieren das `isAllowExternal` rauszunehmen!

Reply:
wenn das PO-User Object überall verfügbar wäre, könnten wir immer mit "not instanceof PO-User" arbeiten. daher dieser flag

Reply:
isInternal